apps/api/src/models/schema.ts

```diff
@@ -44,6 +46,46 @@ export const friendships = mysqlTable('friendship', {
   // TODO: prevent duplicate rows with inviterId and inviteeId swapped
 }));
 
+export const items = mysqlTable('item', {
+  id: int('id').primaryKey().autoincrement(),
+  createdAt: timestamp('created_at').notNull().$defaultFn(() => new Date()),
+  updatedAt: timestamp('updated_at').onUpdateNow(),
+  creatorId: int('creator_id').references(() => users.id, { onDelete: 'set null' }),
+  name: varchar('name', { length: 50 }).notNull(),
+  description: text('description'),
+  imageUrl: varchar('image_url', { length: 255 }),
+  brand: varchar('brand', { length: 50 }),
+  price: decimal('price', { precision: 2 }),
+  isForSale: boolean('is_for_sale').notNull().default(false),
+  isForTrade: boolean('is_for_trade').notNull().default(false),
+  soldAt: timestamp('sold_at'),
+});
```

Comment:
This looks pretty good to me. I'm not sure what else I can add. My preliminary ItemCard uses only a few of these things: title, description, and image. 

In the item modal that opens upon clicking on the itemCard, well display more stuff, but I'm not totally sure what yet, and that can change.

This looks great.